rampwf/score_types/detection.py

```diff
@@ -239,55 +214,99 @@ def scp_single(y_true, y_pred, shape, minipatch=None):
     return score, n_true, n_pred
 
 
-def ospa_single(y_true, y_pred, cut_off=1, minipatch=None):
+def ospa(y_true, y_pred, minipatch=None):
+    """
+    Optimal Subpattern Assignment (OSPA) metric for IoU score.
+
+    This metric provides a coherent way to compute the miss-distance
+    between the detection and alignment of objects. Among all
+    combinations of true/predicted pairs, if finds the best alignment
+    to minimise the distance, and still takes into account missing
+    or in-excess predicted values through a cardinality score.
```

Comment:
Can you add this docstring (also) to the OSPA score type class ?

Reply:
Sure thing

Reply:
Done